gitpractice/javascript/task7.js

```diff
@@ -0,0 +1,15 @@
+/*http://www.codewars.com/kata/partition-on*/
+function partitionOn(pred, items) {
+    var evenNumberArray = items.filter(pred);
+    var oddNumberArray = items.filter(item => {
+        if (pred(item) == false)
+            return true;
+        else
+            return false;
+    })
```

Comment:
for this type of if else statements,
you can use tiernary operator, since it will be more readable.

Not all if else statements should be changed into tiernary, based on the condition and readabililty we need to decide when to use tiernary.